### DISCUSSION TITLE
Possible minor error in the code

### DISCUSSION THREAD

**Discussion by andreamanera** [RosettaDrone/rosettadrone#166] [Q&A]
Hello, 
I've already asked some question about this project as I'm working on something similar to this, anyway I was taking a look at the code when I found this in the MAVLinkReceiver.java file, basically at line 738 (but also some line before) param2 is intended as the yaw angle, while on the MAVLink wiki I found that param2 should be the accept radius, while param4 rapresent the yaw angle, am I correct or am I missing something?
Thanks in advance for your help!
![Untitled](https://github.com/RosettaDrone/rosettadrone/assets/91534516/3b9a7c46-5c0e-47fb-8eae-285390469162)

**Comment by kripper** [comment]
Maybe a bug. 

This seems to be the old waypoints implementation (only supported on some DJI drones).
I don't use that, but the modern MissionManager class.

IMO, this old implementation is too tight dependent and limited by the DJI waypoints structure definition and should be merged in the future into the MissionManager in order to support the complete MAVLink waypoint definitions.

The big difference is that the old code stores the waypoint missions using DJI format (a compatible limited subset of the MAVLink mission information) instead of using MAVLink format.
